### DISCUSSION TITLE
How to Change Default Electron Storage Path?

### DISCUSSION THREAD

**Discussion by sophieminasyan** [eclipse-theia/theia#14136] [General]
Hello everyone,

I’m currently working on an Electron application and I need to customize the default storage path for user data. By default, Electron uses:
    $HOME/.config on Linux
    %APPDATA% on Windows

However, I need to change this location to a different directory. Could someone guide me on how to configure this?

**Comment by msujew** [comment]
Hey @sophieminasyan,

running `<app> --help` on your application will tell you that users can specify the `--electronUserData` argument to change the directory for the electron storage. Alternatively, users can add a `data` folder in the application directory, which will be used to store all data.
